cnf-tests/README.md

```diff
@@ -326,19 +326,19 @@ To enable discovery mode the tests must be instructed by setting the `DISCOVERY_
 Most SRIOV test require the following resources:
 
 - SriovNetworkNodePolicy
-- at least one one with the resource specified by SriovNetworkNodePolicy being allocatable (a resource count of at least 5 is considered sufficient)
+- at least one with the resource specified by SriovNetworkNodePolicy being allocatable (a resource count of at least 5 is considered sufficient)
 
 Some tests have additional requirements:
 
 - an unused device on the node with available policy resource (with link state DOWN and not a bridge slave)
-- a SriovNetworkNodePolicy with a MTU value of 9000
+- an SriovNetworkNodePolicy with an MTU value of 9000
```

Comment:
AFAIK "an" is incorrect here
https://www.quickanddirtytips.com/education/grammar/a-versus-an

Reply:
It is correct because it is read as "EsArAiOVee" (unless I am mistaken), and since the beginning sound is a vowel (e) it should be preceded by "an", just like "an hour" (an AuEr) as explained in the link you sent.

Reply:
@mgourin is right though this is counter-intuitive.